### ISSUE TITLE
Lagring - bug

### ISSUE THREAD

**Issue by AneMarlene** [Artsdatabanken/Fremmedartsbase2023#100]
Når jeg går ut av en art hvor jeg har skriverettigheter, uten å å ha gjort noen endringer inne på arten, dukker denne beskjeden opp. Den dukker opp både når jeg er inne på en art jeg vurderer, og når jeg er inne og ser på en art som har status " Start vurdering". Beskjeden dukker ikke opp om jeg er inne på en art hvor jeg kun har lesertilgang. 

![image](https://user-images.githubusercontent.com/58254444/132189865-a2b1cd3d-1cff-4cf1-ae05-b619224c3366.png)

**Comment by steinho** [comment]
Årsaken er nok at "vurderingen blir endret" under lasting til klienten ... det kan være ubetydelige endringer som trengs for visning (og som ikke lå der fra 2018)
Vi får prøve å identifisere disse - bør være avslørt i json visningen

**Comment by AneMarlene** [comment]
@steinho Ja, vi må finne ut av dette. Jeg fant ingen endringer på lagringstidspunker eller  hvem det var lagret av osv, men vi har jo disse feltene: 

![image](https://user-images.githubusercontent.com/58254444/135420635-c0e810d2-1347-4227-aaf7-e542aa4affc5.png)


Kan det være årsaken, tro?

**Comment by bugdriven** [comment]
Ja, det er årsaken. Skal se om jeg kan fikse dette til på en eller annen måte.  Jeg er litt bekymret for at det gjøres automatiske endringer som er helt nødvendige (i det vurderingen åpnes) og at disse endringene forventes i den ferdige vurderingen. 
Vel, - for at vurderingen skal ferdigstilles så må den uansett lagres så da får vi lagret disse endringene uansett. Men det betyr at alle faktisk må ferdigstille alle vurderinger manuelt!
